README.md

```diff
@@ -70,6 +70,16 @@ us single bullet fire weapons.
 9. Build the VS project
 10. The executables are placed at `../game/hl2.exe` for the client and at `../game/srcds.exe` for the server. Note: this path is outside the repository.
 
+### Building on Linux (To the Makefile)
+**DISCLAIMER:** Make sure you have your distro's build essentials, gcc and g++ multilibs installed. This will vary from distro to distro, so I can't include everything here.
+1. Run `git clone https://github.com/gperftools/gperftools.git -b gperftools-2.0 thirdparty/gperftools-2.0`
+2. CD into `thirdparty/gperftools-2.0` and run`./autogen.sh`
+3. Run `./configure --enable-frame-pointers`
+4. Run `make -j$(nproc)`
+5. Install gcc and g++ multilibs, build essentials. On arch I used the command `sudo pacman -Syu lib32-gcc-libs gcc-libs`. For Ubuntu, according to the Valve Wiki for Source SDK 2013, do `sudo apt-get install build-essential` and `sudo apt-get install gcc-multilib g++-multilib`
+6. ` Run ./creategamesprojects`
+7. If you are going to try to run the makefile, rename it. When it is generated, it has the name `games.sln `. Remove the space at the end to change it to `games.sln`.

```

Comment:
We should say how to run the file; most people won't know how to run an sln on linux.

Reply:
I am just added the things to get to the make file, not actually make it.

Reply:
Do you know why it generates with a space? I think the fix should be to make it generate without it.

Reply:
Nope. Don't know how.